docs/general/metrics.md

```diff
@@ -156,15 +156,15 @@ something should be called `entity.limit`. For example, `system.memory.limit`
 for the total amount of memory on a system.
 
 - **usage** - an instrument that measures an amount used out of a known total
-(**limit**) amount should be called `entity.usage`. For example,
-`system.memory.usage` with attribute `state = used | cached | free | ...` for the
-amount of memory in a each state. Where appropriate, the sum of **usage**
-over all attribute values SHOULD be equal to the **limit**.
-
-  A measure of the amount consumed of an unlimited resource, or of a resource
-  whose limit is unknowable, is differentiated from **usage**. For example, the
-  maximum possible amount of virtual memory that a process may consume may
-  fluctuate over time and is not typically known.
+(**limit**) and has dimensions which add up to the **limit** should be called
```

Comment:
This no longer accounts for "soft limits", like k8s limits.  I think we agreed it was ok for usage + soft limits.  Are you suggesting we remove that use case? if so we need to revisit some k8s / container metrics.